Sources/Networking/Operations/PostReceiptDataOperation.swift

```diff
@@ -71,6 +75,27 @@ class PostReceiptDataOperation: CacheableNetworkOperation {
 
 }
 
+// MARK: - Private
+
+private extension PostReceiptDataOperation {
+
+    func printReceiptData() {
+        do {
+            let receiptData = try JSONEncoder.prettyPrinted.encode(
+                try ReceiptParser.default.parse(from: self.postData.receiptData)
```

Comment:
nitpick: maybe we can have the receipt know how to print itself nicely instead?

Reply:
Sounds good I'll move it there.

Reply:
Actually it already has this:
```swift
var asDict: [String: Any] {
    return [
        "bundleId": bundleId,
        "applicationVersion": applicationVersion,
        "originalApplicationVersion": originalApplicationVersion ?? "<unknown>",
        "opaqueValue": opaqueValue,
        "sha1Hash": sha1Hash,
        "creationDate": creationDate,
        "expirationDate": expirationDate ?? "",
        "inAppPurchases": inAppPurchases.map { $0.asDict }
    ]
}

var description: String {
    return String(describing: self.asDict)
}
```
Maybe I should use that instead?